src/modules/asab-config/MicroservicesContainers/MicroservicesContainer.js

```diff
@@ -36,19 +36,23 @@ export default (props) => {
 	];
 
 	useEffect(() => {
-		ASABConfigAPI.get('/microservices', { params: { p: page, l: limit }})
-			.then(res => {
-				if (!res.data) throw new Error("response.data is undefined or null");
-				if (!res.data.data) throw new Error("response.data.data is undefined or null");
-				setList(res.data.data);
-				setCount(res.data.count);
-			})
-			.catch(e => {
-				console.error(e);
-				props.app.addAlert("warning", t('Failed fetch'));
-			})
+		getMicroservicesList();
 	}, [page, limit]);
 
+	const getMicroservicesList = async () => {
+		try {
+			const response = await ASABConfigAPI.get('/microservices', { params: { p: page, l: limit }});
+
+			if (response.data.result !== "OK") throw new Error(res);
```

Comment:
@titkov49 This `throw new Error(res)` does not seem good to me, `res` is not defined anywhere

Reply:
@Pe5h4 Oh yeah, you're right. Since it's in `try` this also throws error which is catched so I didn't even noticed that